library/src/androidTest/java/com/jmolsmobile/landscapevideocapture/recorder/VideoRecorderTest.java

```diff
@@ -53,10 +55,28 @@
 import static org.mockito.Mockito.times;
 import static org.mockito.Mockito.verify;
 
+
 @SuppressWarnings("deprecation")
-@RunWith(AndroidJUnit4.class)
+@RunWith(value = Parameterized.class)
 public class VideoRecorderTest extends MockitoTestCase {
 
+    private boolean FlashStatus;
+
+    // Inject via constructor
+    public VideoRecorderTest(boolean FlashStatus) {
+        this.FlashStatus = FlashStatus;
+    }
+
+    // name attribute is optional, provide an unique name for test
+    // multiple parameters, uses Collection<Object[]>
+    @Parameterized.Parameters()
```

Comment:
This is overkill, there is no need to check that every interaction with the recorder uses the right value for the flash. Better just to always pass in false, and to add one extra test that validates that true is properly propagated. (or multiple ones if there are different propagations)

Doing it this way basically defocusses every existing test and adds an extra thing for them to implicitly test. That is what I want to avoid: every test should only test for exactly one thing. 

So remove parametrized, always use flash = false and add one (or more) tests to only check if a true value is properly propagated,